src/thor/trace_attributes_action.cc

```diff
@@ -428,8 +428,8 @@ json::MapPtr thor_worker_t::trace_attributes(
         try {
           trip_path = route_match(controller);
           if (trip_path.node().size() == 0)
-            throw;
-        } catch (...) {
```

Comment:
sure yeah! by cleanly i just meant that, with respect to the service worker or the actor, not catching them is the best way to go because it allows them to handle as special cases. so in the service the prime_server code will catch the interrupt exception that it expects and write out a message that says the request was aborted. with the actor, your thread that is calling `actor::route` will be able to (as the tests in this pr shows) catch the special exception that it passes for an interrupt and do something it likes to (in this case it should call cleanup on the actor, i'll change that in a second). if it were to keep catching `...` both the service and the actor would get back a `valhalla_exception_t` which would mask the ability to know that was an interrupt that caused it to fail and not actually failing the computation.